transformations/variable_char_perturbation/README.md

```diff
@@ -0,0 +1,54 @@
+# Variable Character Perturbation 🦎 + 🎮 → 🐍
+This perturbation performs 5 way perturbations on non numeric words, either deletion insertion substitution swapping and duplication, with configurable probability and subset of perturbations to choose from.
+
+For example: Banana -> Banan | Banabna | Banama | Baanna | Bannana
+
+
+Author name: Abhilash Pal
+<br/>
+Author email: abhilash.pal@tum.de / abhilashpal8@gmail.com
+<br/>
+Author Affiliation: Technische Universität München
+
+## What type of a transformation is this?
+This transformation adds random noise to the input word. Based on a predefined probability, words of atleast length 3 are perturbed with either random insertion deletion substitution swapping or duplication. Users can set which of the transformations they want as well.
+Generated transformations display high similarity to the source sentences i.e. the code outputs highly precise generations. 
+
+
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+```python evaluate.py -t VariableCharPerturbation -task TEXT_CLASSIFICATION```
+```model_name = "aychang/roberta-base-imdb"```
+
+The accuracy of a RoBERTa model (fine-tuned on IMDB) on a subset of IMDB sentiment dataset = 96% The accuracy of the same model on the perturbed set = 94%
+
+
+## Previous Work
```

Comment:
To my knowledge there are many character level perturbations (unmerged and merged) previous PRs. It might be a good idea to cite them and mention what this transformation does differently?